js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_en.properties

```diff
@@ -2810,6 +2811,7 @@ clientSessionIdle=Client Session Idle
 push=Push
 targetClaimHelp=Specifies the target claim which the policy will fetch.
 periodicFullSyncHelp=Whether periodic full synchronization of LDAP users to Keycloak should be enabled or not
+client-attributes-condition.tooltip=Client attributes, that will be checked during this condition evaluation. Condition evaluates to true if the client has at least one client attribute with the name and value as the client attributes specified in the configuration.
```

Comment:
According to `CilentAttributesCondition.java`, this condition is true only if attributes of a client includes all of attributes of this condition and matches all values of the attributes of this condition with ones of the attributes of the client.

Reply:
Fixed.